deployment_scripts/appimage/AppRun

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
```

Comment:
Why is AppImage in deployment scripts folder?

Reply:
I could move it to be in the same directory with the `docker/` directory if you think that's a better place.

Reply:
Have you thought about how we're going to integrate this into Jenkins? Booting a new VM for each build might be a bit of an overkill. Is there a way re-build without a fresh VM?

Reply:
For reproducibility, we should be building everything in clean environments. LXC is a lighter-weight alternative to VMs. We may even be able to use a docker as a build environment.